packages/types/index.ts

```diff
@@ -25,6 +25,7 @@ export declare type PDFRequestPayload = {
   scope: string;
   module: string;
   importName?: string;
+  landscape?: boolean;
```

Comment:
Based on the name `PDFRequestPayload` could changes to this type affect the API spec for pdf-generator? Or is this change isolated to frontend-components?